client/src/components/ui/Icons.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+    import { iconsList, Icon } from "./Icons.ts";
+    export let name: string;
+    export let color: "default" | "primary" | "faded" = "default";
```

Comment:
We can probably add an `enum` for this so that the intent is clearer.

```typescript
enum ButtonColor {
    Default,
    Primary,
    Faded,
}
```

Reply:
Fixed by adding enums with string value equivalents and exporting them out to `../types.ts`.